src/PrestaShopBundle/DependencyInjection/AddOnsConfiguration.php

```diff
@@ -37,8 +36,6 @@ public function getConfigTreeBuilder()
         $treeBuilder = new TreeBuilder('prestashop');
         $rootNode = $treeBuilder->getRootNode();
 
-        Tools::refreshCACertFile();
```

Comment:
I'm not sure simply removing this call here is really relevant What I mean is more that the whole `AddOnsConfiguration` should be removed along with its associated config file `app/config/categories.yml` file The 8.0 is supposed to be completely independent from the Addons market place so I guess we just forgot to remove this part.

Also since it was forgotten for 8.0 maybe this should target 8.0.x branch instead? Unless we're worried about the impact and prefer to delay this change into 8.1.0 (both are fine for me).

Reply:
Sorry. The 8.0 is the start of the Split. You can follow the end in #28771.

Reply:
I think better remove this file usage in 8.1 and remove this class in 9.0.